.github/renovate.json

```diff
@@ -152,6 +152,11 @@
       "matchPackagePatterns": ["^@vitest/", "^vitest", "^vite-", "vitest$"],
       "dependencyDashboardApproval": false
     },
+    {
+      "groupName": "emoji-mart npm packages",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@emoji-mart/", "^emoji-mart", "emoji-mart$"],
```

Comment:
```suggestion
      "matchPackagePatterns": ["^@emoji-mart/", "^emoji-mart", "emoji-mart$"]
```

Reply:
Already fixed this and have other pending changes on a local branch.